### ISSUE TITLE
Don't store min+max namespace IDs in badger inverted index keys

### ISSUE THREAD

**Issue by musalbas** [celestiaorg/celestia-node#2617]
This should save around 50% of the storage used by the inverted index. We don't need to put min+max namespace IDs in the inverted index, as they don't contribute to collision resistance. To access a key, you can simply truncate the namespace ID.

**Comment by moldis** [comment]
Is this issue still valid? I dont see any occurrences in the code

**Comment by Wondertan** [comment]
Yes it is. This happens in a subtle way. The good thing is that soon the inverted index will be removed altogether with this issue via new protocol: https://github.com/celestiaorg/celestia-node/issues/2971
